### PR TITLE
Updates to Contribution guide from @meghanjacquot

### PR DESCRIPTION
**Pull request by tghosth**
This was discussed within the working group.

@meghanjacquot prepared the update
@tghosth reviewed and finalized the text

### CODE REVIEW

**Comment by tghosth** [comment]
@jmanico please approve this

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -2,25 +2,44 @@
...
-The current status of the ASVS project is as follows:
+### What is [OWASP](https://owasp.org/)?
+
+The Open Worldwide Application Security Project (OWASP) is a nonprofit organization that works to improve the security of software. It has many programs to work towards this goal. One of those programs is the ASVS.
+
+### What is the [ASVS](https://owasp.org/www-project-application-security-verification-standard/)?
+
+The OWASP Application Security Verification Standard (ASVS) Project provides a basis for testing web application technical security controls and also provides developers with a list of requirements for secure development.
+
+The primary aim of the OWASP Application Security Verification Standard (ASVS) Project is to normalize the range in the coverage and level of rigor available in the market when it comes to performing Web application security verification using a commercially-workable open standard.
```
Comment (jmanico) [inline CONTRIBUTING.md:15]: This is a little wordy. Perhaps "The main goal of the OWASP Application Security Verification Standard (ASVS) Project is to make sure that Web application security checks are consistent and thorough, using an open standard that can be used in the market."

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -2,25 +2,44 @@
...
 
-Please also focus attention on the requirements themselves and **not** on the surrounding text or on the introductory chapters (files 0x01-0x04). This because the text might have become outdated where requirements have changed and also because our goal is to significantly cut down this text anyway in 5.0.
+We will try, on a best efforts basis, to address issues and push changes to the "bleeding edge" master branch within the 5.0 folder **without altering current numbering** and using tagging ([see below](CONTRIBUTING.md#use-tags-to-describe-the-change)) to make changes clearer. This is in order to make it easier to use this branch on an ongoing basis. 
+
+We will no longer be accepting changes to the 4.0 folder which is now fixed at the 4.0.3 release.
 
-### Where do I start?
+Please also focus attention on the requirements themselves and **not** on the surrounding text or on the introductory chapters (files 0x01-0x04). This is because the text might have become outdated where requirements have changed and also because our goal is to significantly cut down this text in version 5.0.
```
Comment (jmanico) [inline CONTRIBUTING.md:25]: cleanup: "Please focus only on the requirements and not on the surrounding text or introductory chapters (files 0x01-0x04). The text might be outdated due to changes in requirements, and we aim to reduce it significantly in version 5.0."

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -2,25 +2,44 @@
...
 
 There are two key types right now:
 
 * [![GitHub issue custom search](https://img.shields.io/github/issues-search?query=repo%3Aowasp%2Fasvs%20is%3Aopen%20is%3Aissue%20label%3A%22Community%20wanted%22&style=flat&label=Community%20Wanted&labelColor=%23BFD4F2&color=grey)](https://github.com/OWASP/ASVS/issues?q=is%3Aopen+is%3Aissue+label%3A%22Community+wanted%22) - These are issues where we would really benefit from more eyes on a particular issue.
 * [![GitHub issue custom search](https://img.shields.io/github/issues-search?query=repo%3Aowasp%2Fasvs%20is%3Aopen%20is%3Aissue%20label%3A%22Community%20needed%22&style=flat&label=Community%20Needed&labelColor=%23fbca04&color=grey)](https://github.com/OWASP/ASVS/issues?q=is%3Aopen+is%3Aissue+label%3A%22Community+needed%22) - These are issues where the relevant items will not get progressed without community input.
 
+
+Please log ideas, issues or questions here: https://github.com/OWASP/ASVS/issues. It’s helpful to share if you have any ideas or if you find any bugs. We may subsequently ask you to open a pull request, https://github.com/OWASP/ASVS/pulls, based on the discussion in the issue.
```
Comment (jmanico) [inline CONTRIBUTING.md:39]: Cleanup "Please log your ideas, issues, or questions here: https://github.com/OWASP/ASVS/issues. Sharing your ideas or reporting bugs is helpful. Based on the discussion, we might ask you to open a pull request here: https://github.com/OWASP/ASVS/pulls."

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -113,10 +132,10 @@ If you are interested in creating a translation, here are some pointers for how
 * Often there is work to do in creating markdown files or updating the translation to keep it up to date with latest changes.
 * We would request that you base your translation on the 4.0/en folder in the master branch as this is now static at the 4.0.3 version.
 * In order to start a translation, please start by forking the ASVS repository.
-    * If you are updating an existing translation which has markdown (just French as at December 2021), you can make modifications to the files in the existing folder based on language code (just /fr as at December 2021).
-    * if you are starting a new markdown translation, take a copy of the /en folder and rename it to the 2 character language code which will be used for the translation. 
+    * If you are updating an existing translation which has markdown (just French as of December 2021), you can make modifications to the files in the existing folder based on language code (just /fr as at December 2021).
```
Comment (jmanico) [inline CONTRIBUTING.md:135]: Changing the French comment and other edits:

    * If you are updating a translation that uses markdown, you can modify the files in the existing folder based on the language code (such as /fr for French).

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -113,10 +132,10 @@ If you are interested in creating a translation, here are some pointers for how
...
+    * If you are starting a new markdown translation, take a copy of the /en folder and rename it to the 2 character language code which will be used for the translation.
```
Comment (jmanico) [inline CONTRIBUTING.md:136]: Smal edit

    * If you are starting a new markdown translation, take a copy of the /en folder and rename it to the 2-character language code used for the translation.

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -113,10 +132,10 @@ If you are interested in creating a translation, here are some pointers for how
...
 * When you have completed the translation, please open a Pull Request against the master ASVS branch and one of the leaders will look at integrating it.
-* The leader will also use the relevant scripts to create the documents from the raw markdown (or you can if you want to save us some trouble  )
-* Finally, the leader will back port the translation into the branch containing ASVS version which was targeted (at this point, presumably v4.0.3)
+* The leader will also use the relevant scripts to create the documents from the raw markdown (or you can if you want to save us some trouble).
```
Comment (jmanico) [inline CONTRIBUTING.md:138]: This is a little strange, perhaps drop the ending?

File: CONTRIBUTING.md [hunk 7]
```diff
@@ -113,10 +132,10 @@ If you are interested in creating a translation, here are some pointers for how
...
+* Finally, the leader will back port the translation into the branch containing the ASVS version which was targeted (at this point, presumably v4.0.3).
```
Comment (jmanico) [inline CONTRIBUTING.md:139]: Need to change the version

File: CONTRIBUTING.md (hunk 1 above)
Comment (tghosth) [reply CONTRIBUTING.md:15]: How about:

> The OWASP Application Security Verification Standard (ASVS) Project provides a basis for testing web application technical security controls and also provides developers with a list of requirements for secure development.
>
> The project aims is to provide a commercially-usable open standard for web application security with a consistent level of coverage and rigor.

File: CONTRIBUTING.md (hunk 2 above)
Comment (tghosth) [reply CONTRIBUTING.md:25]: Done

File: CONTRIBUTING.md (hunk 3 above)
Comment (tghosth) [reply CONTRIBUTING.md:39]: Done

File: CONTRIBUTING.md (hunk 6 above)
Comment (tghosth) [reply CONTRIBUTING.md:138]: Ok

File: CONTRIBUTING.md (hunk 5 above)
Comment (tghosth) [reply CONTRIBUTING.md:136]: Done

File: CONTRIBUTING.md (hunk 4 above)
Comment (tghosth) [reply CONTRIBUTING.md:135]: Done

File: CONTRIBUTING.md (hunk 7 above)
Comment (tghosth) [reply CONTRIBUTING.md:139]: Well for now it will be 4.0.3 until we have a solid 5.0

File: CONTRIBUTING.md (hunk 1 above)
Comment (tghosth) [reply CONTRIBUTING.md:15]:
```suggestion
The OWASP Application Security Verification Standard (ASVS) Project provides a basis for testing web application technical security controls and also provides developers with a list of requirements for secure development.

The project aims is to provide a commercially-usable open standard for web application security with a consistent level of coverage and rigor.
```

**Review by tghosth — COMMENTED** [review]
@jmanico I added some responses

File: CONTRIBUTING.md [hunk 8]
```diff
@@ -2,25 +2,44 @@
...
+## Introduction
 
-The current status of the ASVS project is as follows:
+### What is [OWASP](https://owasp.org/)?
+
+The Open Worldwide Application Security Project (OWASP) is a nonprofit organization that works to improve the security of software. It has many programs to work towards this goal. One of those programs is the ASVS.
+
+### What is the [ASVS](https://owasp.org/www-project-application-security-verification-standard/)?
+
+The OWASP Application Security Verification Standard (ASVS) Project provides a basis for testing web application technical security controls and also provides developers with a list of requirements for secure development.
```
Comment (tghosth) [inline CONTRIBUTING.md:13]:
```suggestion
The OWASP Application Security Verification Standard (ASVS) Project provides a basis for testing web application technical security controls and also provides developers with a list of requirements for secure development.
```
@jmanico

**Review by jmanico — APPROVED** [review]

**Pull request merged**